inst/templates/ad_adeg.R

```diff
@@ -227,7 +227,7 @@ adeg <- adeg %>%
 ## Derive baseline flags ----
 adeg <- adeg %>%
   # Calculate BASETYPE
-  derive_var_basetype(
+  derive_basetype_records(
     basetypes = rlang::exprs(
```

Comment:
shall we drop `rlang::` so match ad_advs.R template?

Reply:
Updated